src/Widgets/TickListWidget/TickListWidgetStyles.scss

```diff
@@ -1,3 +1,8 @@
+@import "../../assets/stylesheets/bootstrap/mixins/clearfix";
+@import "../../assets/stylesheets/fontawesome/variables";
+@import "../../assets/stylesheets/fontawesome/core";
```

Comment:
* [ ] Tha FA core is not widget specific here. Would this belong to a global CSS instead?

Reply:
This is needed to make `@extend .fa;` in line 25 work. Otherwise you'll get the following scss error:

```
ModuleBuildError: Module build failed (from ../node_modules/sass-loader/lib/loader.js):

  @extend .fa;
         ^
      ".tick-list li:before" failed to @extend ".fa".
The selector ".fa" was not found.
Use "@extend .fa !optional" if the extend should be able to fail.
```